Project/PL2/PL2/Controller/HomeVC.swift

```diff
@@ -3101,6 +3104,23 @@ class HomeVC: UIViewController , UICollectionViewDelegate, UICollectionViewDataS
             }
         }
     }
+     
+    func prepareImageView(x: Double, y: Double, isWhiteImage: Bool) {
+          //image for whiote color grid
+          var yourImage: UIImage = UIImage(named: "canvas") ?? UIImage()
+          if !isWhiteImage {
+              //image for numbered grid
+              yourImage = UIImage(named: "yarn") ?? UIImage()
+          }
+          let pixelBoxFrame = CGRect(x: x, y: y, width: squareWidth, height: squareWidth)
+          // Create a UIImageView for the fillImage
+          let imageView = UIImageView(frame: pixelBoxFrame)
```

Comment:
I think adding ImageView to each box may cause memory issue? @vivek-srivastava-iphtech

Reply:
test on a complex image of 100x100 and compare with only colo fill, how many memory consumption is increasing?